delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/wiring/OrganizationsModule.scala

```diff
@@ -33,15 +30,16 @@ object OrganizationsModule extends ModuleDef {
     (
         config: AppConfig,
         scopeInitializations: Set[ScopeInitialization],
-        clock: Clock[UIO],
+        clock: Clock[IO],
         uuidF: UUIDF,
-        xas: Transactors
+        xas: Transactors,
+        contextShift: ContextShift[IO]
     ) =>
       OrganizationsImpl(
         scopeInitializations,
         config.organizations,
         xas
-      )(clock, uuidF)
+      )(clock, uuidF, contextShift)
```

Comment:
Not related to this PR but it feels that we should rework how we run those scope initialization.
It is not up to orgs/projects to know how they should be run.
One other thing is that it introduces circular dependencies between orgs and the modules with the scope initialization implementations